src/rgw/rgw_op.cc

```diff
@@ -4376,10 +4376,11 @@ void RGWPutObj::execute(optional_yield y)
   }
 
   tracepoint(rgw_op, processor_complete_enter, s->req_id.c_str());
+  const req_context rctx{this, s->yield, s->trace.get()};
```

Comment:
this doesn't compile when `WITH_JAEGER=OFF`:
```
src/rgw/rgw_op.cc: In member function ‘virtual void RGWPutObj::execute(optional_yield)’:
src/rgw/rgw_op.cc:4379:51: error: ‘class jspan’ has no member named ‘get’
 4379 |   const req_context rctx{this, s->yield, s->trace.get()};
      |                                                   ^~~
```

Reply:
wonder if we need the full `std::unique_ptr` interface (e.g. `reset()`, `swap()`)?

Reply:
fixed, now it compiles with `WITH_JAEGER=OFF`